conf/messages.cy

```diff
@@ -335,195 +335,183 @@ jobseekersAllowance.agent.jsa = Lwfans Ceisio Gwaith
+ukInterest.noInterest = Na, ni chefais unrhyw incwm o log yn y DU
+ukInterest.agent.fromUkBanks = Llog o fanciau a chymdeithasau adeiladu yn y DU
+ukInterest.agent.fromUkTrustFunds = Llog o gronfeydd ymddiriedolaeth neu fondiau yn y DU
+ukInterest.agent.fromGiltEdged= Llog o warantau gilt neu warantau incwm cronedig
+ukInterest.agent.noInterest = Na, ni chafodd fy nghleient unrhyw incwm o log yn y DU
+ukInterest.error.required = Dewiswch a gawsoch incwm o log yn y DU, neu dewiswch ‘Na, ni chefais unrhyw incwm o log yn y DU’
+ukInterest.agent.error.required = Dewiswch a gafodd eich cleient incwm o log yn y DU, neu dewiswch ‘Na, ni chafodd fy nghleient unrhyw incwm o log yn y DU’
+
+ukInsuranceGains.title = Enillion yswiriant yn y DU
+ukInsuranceGains.agent.title = Enillion yswiriant yn y DU
+ukInsuranceGains.legend = A gawsoch enillion o bolisïau yswiriant yn y DU neu gyfrifon ISA sydd wedi’u dirymu?
+ukInsuranceGains.agent.legend = A gafodd eich cleient enillion o bolisïau yswiriant yn y DU neu gyfrifon ISA sydd wedi’u dirymu?
+ukInsuranceGains.summary = Deall enillion yswiriant
+ukInsuranceGains.p1 = Mae’r adegau mwyaf cyffredin y byddwch yn gwneud enillion fel a ganlyn:
+ukInsuranceGains.p2 = Os yw’r cyfrifiadau sy’n dilyn y digwyddiadau hyn yn dangos bod ennill wedi’i wneud, dylai’ch yswiriwr anfon tystysgrif digwyddiad trethadwy atoch sy’n dangos yr ennill.
+ukInsuranceGains.p3 = Nid yw contractau blwydd-daliadau bywyd yn cynnwys blwydd-daliadau pensiwn.
+ukInsuranceGains.agent.p1 = Mae’r adegau mwyaf cyffredin y bydd eich cleient yn gwneud enillion fel a ganlyn:
+ukInsuranceGains.agent.p2 = Os yw’r cyfrifiadau sy’n dilyn y digwyddiadau hyn yn dangos bod ennill wedi’i wneud, dylai yswiriwr eich cleient anfon tystysgrif digwyddiad trethadwy ato sy’n dangos yr ennill.
+ukInsuranceGains.agent.p3 = Nid yw contractau blwydd-daliadau bywyd yn cynnwys blwydd-daliadau pensiwn.
+ukInsuranceGains.details.bullet1 = cafwyd arian parod neu fudd-daliadau eraill wrth ildio polisi’n llawn neu’n rhannol
+ukInsuranceGains.details.bullet2 = os gwnaeth polisi aeddfedu neu ddod i ben oherwydd marwolaeth y bywyd a yswiriwyd
+ukInsuranceGains.details.bullet3 = gwerthwyd neu neilltio polisi yn y DU, neu ran o bolisi, ar gyfer gwerth.
+ukInsuranceGains.details.bullet4 = os oedd y polisi yn Fond Portffolio Personol, hyd yn oed os nad oedd yr yswiriwr wedi talu arian parod na budd-daliadau eraill yn ystod y flwyddyn
+ukInsuranceGains.lifeInsurance = Yswiriant Bywyd
+ukInsuranceGains.lifeAnnuity = Blwydd-dal Bywyd
+ukInsuranceGains.capitalRedemption = Adbrynu Cyfalaf
+ukInsuranceGains.voidedISA = ISA sydd wedi’i ddirymu – polisi a ganslwyd gan reolwr eich ISA
+ukInsuranceGains.agent.lifeInsurance = Yswiriant Bywyd
+ukInsuranceGains.agent.lifeAnnuity = Blwydd-dal Bywyd
+ukInsuranceGains.agent.capitalRedemption = Adbrynu Cyfalaf
+ukInsuranceGains.agent.voidedISA = Canslwyd gan reolwr ISA eich cleient)
```

Comment:
Could we please double check this line, I think it is missing the start of the message.
The full message in English is: `Voided ISA (a policy cancelled by your client’s ISA manager)`

Line 470 seems to be missing a translation of: `Voided ISA (a policy ` 

Maybe it should be in the format of the individual message.